src/gam.py

```diff
@@ -818,32 +818,32 @@ def doGAMCheckForUpdates(forceCheck=False):
   except (urllib2.HTTPError, urllib2.URLError):
     return
 
-def doGAMVersion(checkForCheck=True):
+def doGAMVersion(checkForArgs=True):
   force_check = False
   simple = False
-  i = 2
-  while i < len(sys.argv):
-    myarg = sys.argv[i].lower().replace(u'_', u'')
-    force_check = True
-    if myarg == u'check':
-      force_check = True
-      i += 1
-    elif myarg == u'simple':
-      simple = True
-      i += 1
-    else:
-      print u'ERROR: %s is not a valid argument for "gam version"' % sys.argv[i]
-      sys.exit(2)
+  if checkForArgs:
+    i = 2
+    while i < len(sys.argv):
+      myarg = sys.argv[i].lower().replace(u'_', u'')
+      if myarg == u'check':
+        force_check = True
+        i += 1
+      elif myarg == u'simple':
+        simple = True
+        i += 1
+      else:
+        print u'ERROR: %s is not a valid argument for "gam version"' % sys.argv[i]
+        sys.exit(2)
   if simple:
-    sys.stdout.write(__version__)
-    sys.exit(0)
-  import struct
-  version_data = u'GAM {0} - {1}\n{2}\nPython {3}.{4}.{5} {6}-bit {7}\ngoogle-api-python-client {8}\n{9} {10}\nPath: {11}'
-  print version_data.format(__version__, GAM_URL, __author__, sys.version_info[0],
-   sys.version_info[1], sys.version_info[2], struct.calcsize(u'P')*8,
-   sys.version_info[3], googleapiclient.__version__, platform.platform(),
-   platform.machine(), GM_Globals[GM_GAM_PATH])
-  if checkForCheck or force_check:
+    print __version__
```

Comment:
print adds a newline here which we don't work. Simple is meant for (tba) upgrade shell scripts to know current version.